This Python package is not available as a binary for all architectures and building it
requires TensorFlow to be installed, i.e. there is a cyclic dependency.
As it is not actually required (but optional) remove it from `REQUIRED_PACKAGES`.
See https://github.com/tensorflow/tensorflow/issues/56636

Author: Alexander Grund (TU Dresden)

diff --git a/tensorflow/tools/pip_package/setup.py b/tensorflow/tools/pip_package/setup.py
index a9ef0ee00b2..bcac1449099 100644
--- a/tensorflow/tools/pip_package/setup.py
+++ b/tensorflow/tools/pip_package/setup.py
@@ -106,7 +106,6 @@ REQUIRED_PACKAGES = [
     # Keras release is not backward compatible with old tf release, and we have
     # to make the version aligned between TF and Keras.
     'keras >= 2.7.0rc0, < 2.8',
-    'tensorflow-io-gcs-filesystem >= 0.21.0',
 ]
 
 
